PREFLIGHT.md

```diff
@@ -0,0 +1,37 @@
+# Optimization 1: Multihost recommended network settings
+We included all the recommended network settings in [rto_setup.sh](https://github.com/google/maxtext/blob/main/rto_setup.sh). 
+
+[preflight.sh](https://github.com/google/maxtext/blob/main/preflight.sh) will help you apply them based on GCE or GKE platform.
+
+Before you run ML workload on Multihost with GCE or GKE, simply apply `bash preflight.sh PLATFORM=[GCE or GKE]` to leverage the best DCN network performance.
+
+Here is an example for GCE:
+```
+bash preflight.sh PLATFORM=GCE && python3 MaxText/train.py MaxText/configs/base.yml run_name=$YOUR_JOB_NAME
+```
+
+Here is an example for GKE:
+```
+bash preflight.sh PLATFORM=GKE && python3 MaxText/train.py MaxText/configs/base.yml run_name=$YOUR_JOB_NAME
+```
+
+# Optimization 2: Numa binding
+NUMA binding is recommended for enhanced performance, as it reduces memory latency and maximizes data throughput, ensuring that your high-performance applications operate more efficiently and effectively.
```

Comment:
Can you make it clear this is v4 and v5p only?

Reply:
Done.